shotgun_api3/lib/mockgun/mockgun.py

```diff
@@ -361,8 +377,9 @@ def create(self, entity_type, data, return_fields=None):
         
         row = self._get_new_row(entity_type)
         
-        self._update_row(entity_type, row, data)        
+        self._update_row(entity_type, row, data)
         row["id"] = next_id
+        row["created_at"] = datetime.datetime.now()
```

Comment:
`created_at` can be set by a `shotgun.create` call. You should test for `"created_at" in data` first.